src/VisualStudio/Core/Def/StackTraceExplorer/StackFrameViewModel.cs

```diff
@@ -19,9 +19,19 @@
 using Microsoft.VisualStudio.Text.Classification;
 using Roslyn.Utilities;
 using System.Diagnostics;
+using System.Collections.Immutable;
+using Microsoft.CodeAnalysis.EmbeddedLanguages.Common;
+using Microsoft.CodeAnalysis.EmbeddedLanguages.StackFrame;
+using System.Text;
+using System.Linq;
+using Microsoft.CodeAnalysis.PooledObjects;
```

Comment:
#sortallthethings

Reply:
my kingdom for sort/remove usings on save :/